index.html

```diff
@@ -48,5 +48,162 @@ <h1 class="main-section__introduction">I'm Mariana Glad to see you</h1>
+          </div>
+        </div>
+      </div>
+
+      <div class="project-items">
+        <img src="images/portfolio2.png" alt="Project photo" />
+        <div class="project-info">
+          <h2>Multi-Post Stories</h2>
+          <div class="project-details">
+            <span class="project-detail">CANOPY</span>
+            <ul class="project-stack">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="project-tech">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <div class="project-button">
+            <a href="">See Project</a>
+          </div>
+        </div>
+      </div>
+      <div class="project-items">
+        <img src="images/portfolio1.png" alt="Project photo" />
+        <div class="project-info">
+          <h2>Tonic</h2>
+          <div class="project-details">
+            <span class="project-detail">CANOPY</span>
+            <ul class="project-stack">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="project-tech">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <div class="project-button">
+            <a href="">See Project</a>
+          </div>
+        </div>
+      </div>
+      <div class="project-items">
+        <img src="images/portfolio1.png" alt="Project photo" />
+        <div class="project-info">
+          <h2>Multi-Post Stories</h2>
+          <div class="project-details">
+            <span class="project-detail">CANOPY</span>
+            <ul class="project-stack">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="project-tech">
+            <span>html</span>
+            <span>css</span>
+            <span>javascript</span>
+          </div>
+          <div class="project-button">
+            <a href="">See Project</a>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-section">
+      <h2>About<br />Myself</h2>
+
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div>
+        <a href="#"><p>let's connect</p></a>
```

Comment:
- [ ] Please note that `let's connect` is not a link, its a title for the social links below it, so instead of wrapping it in a `<a>` tag it should a `<hx>` tag, kindly fix it